src/door.py

```diff
@@ -1,12 +1,19 @@
 """"""
 from room import Room
+import random
 
 
 class Door:
     """"""
 
-    def set_numbers(self) -> None:
-        """"""
+    def __init__(self, next_room_x: int, next_room_y: int, room_trap: bool):
+        self._next_room_coordinates = next_room_x, next_room_y
+
+        if room_trap is True:  # temporary solution
```

Comment:
Why not just `if room_trap:`?

Reply:
My fault. Sometimes I write in autopilot mode and skips simplifications.